docs/geedocs/5.1.3/answer/1066449.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html> <html> <head> <meta charset="UTF-8"/>
 <title>Custom folders</title>
-<link rel="stylesheet" href="../css/styles.css" type="text/css" media="all" /> <link rel="stylesheet" href="../css/containers.css" type="text/css" media="all" /> </head> <body>
+<link rel="stylesheet" href="../../css/styles.css" type="text/css" media="all" /> <link rel="stylesheet" href="../../css/containers.css" type="text/css" media="all" /> </head> <body>
```

Comment:
Never mind on the never mind.  The css directory could perhaps be shared between all the releases, but the art directory should probably be specific for the release because the images could change between releases.  I'm open to discussion on that, though.